js/search.js

```diff
@@ -36,7 +48,30 @@ function(ytData, playlist, template, $) {
     }
 
     var displaySearchResults = function(){
-        $("#search_res").empty()
+        $("#search_res").empty();
+        $("#search_res").append("<li><h6 class='ti'>Playlist</h6></li>");
+        if(searchPlaylist===null){
+            $("#search_res").append("<p>Nothing found</p>");
+        }else{
+            for(song in searchPlaylist){
+                var vidrow=template.getVidRow(searchPlaylist[song]);
+                $(vidrow).addClass("item-search");
+                $(vidrow).on("click",function(){
+                    //console.log('clicked '+this.id);
+                    var hashId="#playlist > #"+this.id;
+                    $("#playlist").animate({
+                            scrollTop : $(hashId).offset().top
+                        }, 2000);
+                    $(hashId).css({'background-color':'#56BD81'});
```

Comment:
Could you instead use Materialize colour classes, instead of modifying the style. It will maintain consistency. http://materializecss.com/color.html